### ISSUE TITLE
Improve discoverability of Contribution docs 

### ISSUE THREAD

**Issue by gked** [pocketnetteam/pocketnet.core#497]
This project, as many open-source projects that don't have the muscle of some investment firm backing it, has to rely on the community. 
Community helps with feedback, stimulates organic growth, creates and enforces the rules in the social network, and finally, it helps with building a product. 
So those who are willing to help in various capacities should be exposed to information on how they could contribute as soon as they join the Bastyon.

There are a coulpe of ways that come to mind:

- A message with links, as soon as a user joins, similar to what people receive today about the secure messenger. 
- A new user is automatically enrolled to follow one of the Bastyon's help team accounts. Bastyon's help team periodically circulates contribution materials through posts.
- It would be useful if the Bastyon team writes some how-tos or shared various resources, using the long article format and reposting them, periodically.

The bottom line - the discoverability requirement has to be met.

Types of contributions
The initial document should include how volunteers could help Bastyon, e.g. how they can help promote it.

Some examples for help docs on how-to:

- How to file a bug/complain/feature request, etc.
- How they can contribute to the development of the project. There would be a link to the contributions.md.
- Promotion of Bastyon with links to documents with how-tos and best practices.
- Art, Writing, Localization, Author migration from other platforms, etc.

**Comment by andyoknen** [comment]
https://opensource.guide
